src/database/core/util/util.ts

```diff
@@ -114,7 +114,7 @@ export const buildLogMessage_ = function(var_args) {
  * Use this for all debug messages in Firebase.
  * @type {?function(string)}
  */
-export var logger = console.log.bind(console);
+export var logger = null;
```

Comment:
Yes, the value is originally `null`. I set it because I was using the logging everywhere.